internal/characters/tighnari/stats.go

```diff
@@ -0,0 +1,211 @@
+package tighnari
+
+var (
+	attack = [][][]float64{
+		//1
+		{
+			{
+				0.4463,
+				0.4827,
+				0.519,
+				0.5709,
+				0.6072,
+				0.6488,
+				0.7058,
+				0.7629,
+				0.82,
+				0.8823,
+				0.9446,
+				1.0069,
+				1.0691,
+				1.1314,
+				1.1937,
+			},
+		},
+		//2
+		{
+			{
+				0.4197,
+				0.4538,
+				0.488,
+				0.5368,
+				0.571,
+				0.61,
+				0.6637,
+				0.7174,
+				0.771,
+				0.8296,
+				0.8882,
+				0.9467,
+				1.0053,
+				1.0638,
+				1.1224,
+			},
+		},
+		//3
+		{
+			{
+				0.2645,
+				0.286,
+				0.3075,
+				0.3383,
+				0.3598,
+				0.3844,
+				0.4182,
+				0.452,
+				0.4859,
+				0.5228,
+				0.5597,
+				0.5965,
+				0.6334,
+				0.6704,
+				0.7072,
+			},
+			{
+				0.2645,
+				0.286,
+				0.3075,
+				0.3383,
+				0.3598,
+				0.3844,
+				0.4182,
+				0.452,
+				0.4859,
+				0.5228,
+				0.5597,
+				0.5965,
+				0.6334,
+				0.6704,
+				0.7072,
+			},
+		},
+		//4
+		{
+			{
+				0.6863,
+				0.7421,
+				0.798,
+				0.8778,
+				0.9337,
+				0.9975,
+				1.0853,
+				1.1731,
+				1.2608,
+				1.3566,
+				1.4524,
+				1.5481,
+				1.6439,
+				1.7396,
+				1.8354,
+			},
+		},
+	}
+
+	aim = []float64{
```

Comment:
this is the multiplier for the phys aimed shot ("Aimed Shot"), when you should be actually using the values for his dendro aimed shot ("Level 1 Aimed Shot"). in my bow ca pr i use the `aim` slice for phys and `fullaim` slice for the element arrow. so please add a fullaim slice and adjust the multiplier in aimed.go